src/StatusQ/Popups/StatusPopupMenu.qml

```diff
@@ -19,6 +19,8 @@ Menu {
     property var subMenuItemIcons: []
     property var openHandler
     property var closeHandler
+    property string currentImg
+    property string currentIdColor
```

Comment:
It doesn't seem to be obvious why these ones are around.  Is this fixing some bug?

Reply:
no. those are exposed so that it can be set in the Button that opens the popup menu.

Reply:
Got it!

We should come up with better names though. Especially for `currentIdColor`. How about `currentItemColor` and then we can also go with `currentItemImageSource`?

Reply:
changed to `currentOptionItemColor` and `currentOptionItemImgSource`

Reply:
Great